universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam.csl

```diff
@@ -149,8 +149,8 @@
               </else>
             </choose>
           </if>
-          <else-if type="thesis">
-            <text variable="genre" suffix="."/>
+          <else-if type="thesis"><!--Note digunakan untuk mensitasi skripsi, thesis dan disertasi-->
+            <text variable="note" suffix="."/>
```

Comment:
why this change? The note field can contain all types of information, whereas genre ("Genre" in Mendeley, "Type" in Zotero) is intended for describing the type of thesis.